### PR TITLE
disable JIRA creation, provide override

### PR DESCRIPTION
**Pull request by laurafitzgerald**
## JIRA
https://issues.redhat.com/browse/DEL-62

## Changes
- Updates relating to amq-online script additions
- Disable JIRA creation for now and provide override option

## Verification
AMQ changes - Have a look at [this pr](https://github.com/integr8ly/integreatly-operator/pull/463) which was created by [this run](https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Delorean%202.0/job/delorean-2-ews/job/amq-online/job/lfdiscovery/4/console) of the pipeline based on DEL-62 branch of integreatly-operator
JIRA - Search for `[INFO] JIRA issue creation is disabled for now.` in [this output 
](https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Delorean%202.0/job/delorean-2-ews/job/amq-online/job/lfdiscovery/4/console
)
JIRA creation - JIRA was created here: https://issues.redhat.com/browse/INTLY-5832 Feel free to delete it after the verification.

### CODE REVIEW

File: jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile [hunk 1]
```diff
@@ -185,7 +187,7 @@ pipeline {
                     String title = "[WIP] ${productName} update"
                     String body = "Created automatically by jenkins"
                     String[] defaultPRLabels = ["do-not-merge/work-in-progress"]
-                    if (params.dryRun) {
+                    if (params.dryRun && doPush) {
```
Comment (StevenTobin) [inline jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile]: This seemed to be causing me issues. It seems to be interpreting it as a function https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Delorean%202.0/job/delorean-2-ews/job/fuse-online/job/discovery/11/console

Comment (laurafitzgerald) [reply jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile]: Hmm. Not sure why this change got added in this pr, I'll remove it.

File: jobs/delorean/2.0/ews/amq-online/discovery.yaml [hunk 2]
```diff
@@ -15,8 +15,8 @@
       - ci-trigger:
           override-topic: Consumer.rh-jenkins-ci-plugin.8cfe81de-2e51-4b77-96a9-191a1a559fb1.VirtualTopic.eng.brew.build.complete
           message-checks:
-          - field: '$.info.package_id'
-            value: '74050'
+            - field: '$.info.package_id'
+              value: '73640'
```
Comment (mikenairn) [inline jobs/delorean/2.0/ews/amq-online/discovery.yaml]: Is this the switch the correct AMQ brew build?

File: jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile [hunk 3]
```diff
@@ -120,7 +121,7 @@ pipeline {
             steps {
                 script {
                     gitCommitWhenChanges("Added New image Manifests") { msgs ->
-                        sh "scripts/process-image-manifests ${image} ${productName} ${productOperator}"
+                        sh "scripts/process-image-manifests ${image} ${productName}"
```
Comment (mikenairn) [inline jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile]: This changed merged back to delorean-ews?

Comment (laurafitzgerald) [reply jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile]: Yes, we don't use it anymore but just noticed it needs removing here: https://github.com/integr8ly/integreatly-operator/blob/delorean-ews/scripts/process-image-manifests#L28 you can see that's the only entry for that env var

File: jobs/delorean/2.0/ews/amq-online/discovery.yaml (hunk 2 above)
Comment (laurafitzgerald) [reply jobs/delorean/2.0/ews/amq-online/discovery.yaml]: Yep exactly, the one that was mentioned here: https://issues.redhat.com/browse/DEL-62?focusedCommentId=13976209&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-13976209

File: jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile (hunk 3 above)
Comment (laurafitzgerald) [reply jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile]: Applied here: https://github.com/integr8ly/integreatly-operator/pull/468/commits/15301e2f70e038bf8c5eacbf0fe1c5d8fa132622

**Pull request merged**